cli/azd/pkg/executil/executil.go

```diff
@@ -197,10 +198,29 @@ type RunArgs struct {
 	EnrichError bool
 }
 
+type redactData struct {
+	matchString   *regexp.Regexp
+	replaceString string
+}
+
 func (rr RunResult) String() string {
 	return fmt.Sprintf("exit code: %d, stdout: %s, stderr: %s", rr.ExitCode, rr.Stdout, rr.Stderr)
 }
 
+func redactSensitiveData(msg string) string {
+	var regexpRedactRules = map[string]redactData{
+		"access token": {
+			regexp.MustCompile("\"accessToken\": \".*\""),
+			"\"accessToken\": \"redact to prevent sensitive data\"",
```

Comment:
For correct grammar, I think this sentence would need to say "redact**ed** to prevent sensitive data **leakage**". To avoid the long sentence, I think just printing `<redacted>` would suffice 

```suggestion
			"\"accessToken\": \"<redacted>\"",
```